DmitriyStavitscky/src/main/java/ua/artcode/stream_tasks/Tasks.java

```diff
@@ -0,0 +1,106 @@
+package ua.artcode.stream_tasks;
+
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Tasks {
+
+    private List<User> users;
+
+    public Tasks(List<User> users) {
+        this.users = users;
+    }
+
+    // find users(women) that live in kiev and age > 25 age < 30 sorted by age
+    public List<User> findUsersBySexCityAge(String sex, String city, int ageFrom, int ageTo) {
+
+        return users.stream()
+                .filter(u -> u.getSex().equals(sex))
+                .filter(u -> u.getCity().equalsIgnoreCase(city))
+                .filter(u -> u.getAge() >= ageFrom && u.getAge() <= ageTo)
+                .sorted(Comparator.comparingInt(User::getAge))
+                .collect(Collectors.toList());
+    }
+
+    // get first three top users(the most high salary) sorted by salary
+    public List<User> getTop(int count) {
+
+        return users.stream()
+                .sorted(Comparator.comparingDouble(User::getSalary))
+                .skip(users.size() - count)
+                .collect(Collectors.toList());
+    }
+
+    // sum all salaries of Odessa's users, user name should start with letter 'A'
+    public double sumAllSalaries(String city, String firstSymbolOfName) {
+
+        return users.stream()
+                .filter(u -> u.getCity().equals(city) &&
+                        u.getName().startsWith(firstSymbolOfName))
+                .mapToDouble(User::getSalary).sum();
+    }
+
+    // Add 5%(from salary) to every user, print with next patter 'id=,name=,salary='
+    public void addSalaryAndPrint(int percent) {
```

Comment:
My suggestion is not to keep several logic in one method
First method = addSalary
Second = print all workers

You will be able to compose in any variations as you need